app/views/courses/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<table id="courses" class="datatable display table table-striped table-bordered" >
+<table id="courses" class="datatable table table-striped table-bordered" >
```

Comment:
In other places you have changed this to `generic_datatable`. Should this be the same ? What is the difference ?